### PR TITLE
Make reek happy

### PR DESCRIPTION
**Pull request by TheFehr**

### CODE REVIEW

File: lib/helpers/ynab/error_helper.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Helpers
+  module YNAB
+    module ErrorHelper
+      def build_error_message(error, entries)
+        'ERROR:'\
```
Comment (lukasbischof) [inline lib/helpers/ynab/error_helper.rb]: what about HEREDOC?

File: lib/models/ynab/transaction.rb [hunk 2]
```diff
@@ -15,13 +15,13 @@ class Transaction
...
       validate :payee_info?
 
-      def initialize(account_id, date, payee_name, payee_id, amount, memo)
-        @account_id = account_id
-        @date = date
-        @payee_name = payee_name
-        @payee_id = payee_id
-        @amount = amount
-        @memo = memo
+      def initialize(*data)
```
Comment (lukasbischof) [inline lib/models/ynab/transaction.rb]: Why splatting?

File: .reek.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+detectors:
+  IrresponsibleModule:
+    enabled: false
+  UncommunicativeVariableName:
+    accept:
+      - e
+
+
```
Comment (lukasbischof) [inline .reek.yml]: lines empty

File: lib/helpers/ynab/error_helper.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Helpers
+  module YNAB
+    module ErrorHelper
+      def build_error_message(error, entries)
+        <<-ERROR_MESSAGE
```
Comment (lukasbischof) [inline lib/helpers/ynab/error_helper.rb]:
```suggestion
        <<~ERROR_MESSAGE
```
And indent rest

**Review by lukasbischof — APPROVED** [review]
What a colossal Pull Request

**Comment by TheFehr** [comment]
@lukasbischof how does it look now?

**Comment by lukasbischof** [comment]
Now, it looks astonishingly good 😮

**Pull request merged**